content/shmem_malloc.tex

```diff
@@ -31,16 +31,16 @@
     
     The \FUNC{shmem\_free} routine causes the block to which \VAR{ptr} points to be
     deallocated, that is, made available for further allocation.  If \VAR{ptr} is a
-    null pointer, no action occurs. 
+    \const{NULL} pointer, no action occurs.
```

Comment:
The original text was correct. This text should read: "If _var_ is a null pointer, no action occurs."